### PR TITLE
Add Github action to manage to-triage label

### PR DESCRIPTION
**Pull request by ov7a**
1. Add triage label to issues without any labels
2. Remove triage label from issue closed as "not planned"
3. Add triage label to completed issue without milestone

Can be tested [here](https://github.com/ov7a/issue-management-test/issues)

### CODE REVIEW

File: .github/workflows/closed_issue_labels.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        with:
+          labels: to-triage
+
+  add_triage_label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
```
Comment (hegyibalint) [inline .github/workflows/closed_issue_labels.yml:28]: ❓Same as above

**Review by hegyibalint — CHANGES_REQUESTED** [review]

File: .github/workflows/closed_issue_labels.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    types: closed
+
+jobs:
+  remove_triage_label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (ov7a) [reply .github/workflows/closed_issue_labels.yml]: Indeed, missed that. Thanks!

**Review by hegyibalint — APPROVED** [review]

**Review by eskatos — APPROVED** [review]
:+1:

**Comment by ov7a** [comment]
@bot-gradle test and merge

**Pull request merged**